lib/src/mobile_scanner_controller.dart

```diff
@@ -84,6 +86,17 @@ class MobileScannerController extends ValueNotifier<MobileScannerState> {
   /// Defaults to the back-facing camera.
   final CameraFacing facing;
 
+  /// The lens type for the camera.
+  ///
+  /// This allows selection between normal, wide, and zoom lenses on devices
+  /// with multiple cameras.
+  ///
+  /// Defaults to [CameraLensType.any], which uses the first available camera
+  /// for the given [facing] direction.
+  ///
+  /// Currently only supported on iOS 13+ and Android.
```

Comment:
```suggestion
  /// Currently only supported on iOS and Android.
```

IIRC Flutter's minimum iOS version will soon be 13.0